internal/processing/ory/jwt_validator.go

```diff
@@ -0,0 +1,57 @@
+package ory
```

Comment:
I would move this to validation package

Reply:
If we move it to validation then the visibility would need to change and it could be used outside of the intended scope. 
That's why I'd rather think about a different structure for all packages as imho validation is part of the processing and might be better a sub-package and with a similar sub-package for each specific reconciliation e.g. ory/validation, istio/validation, processing/validation.

I think we also need to improve the implementation of validation at some point.

What do you think @barchw ?

Reply:
Yes that would be ok, or validation/ory